docs/en/install-upgrade/breaking.asciidoc

```diff
@@ -31,9 +31,9 @@ Upgrade Assistant provides information about which {dfeeds} need to be updated.
 coming[7.4.0]
 
 This list summarizes the most important breaking changes in APM. 
-For the complete list, go to {apm-server-ref-70}/breaking-changes.html[APM Server 7.0 breaking changes].
+For the complete list, go to {apm-server-ref}/breaking-changes.html[APM Server 7.0 breaking changes].
```

Comment:
In my opinion, it seems a bit strange to call these 7.0 breaking changes in this context:
```suggestion
For the complete list, go to {apm-server-ref}/breaking-changes.html[APM Server breaking changes].
```